build.sh

```diff
@@ -1,16 +1,16 @@
-#!/bin/sh
-case "$1" in
+#!/bin/sh -eu
```

Comment:
I get failure on "HTTP_PROXY: unbound variable" when building locally due to  the -u flag
Possibly we should remove the proxy build args but either way the -e should be enough I guess

Reply:
Ah, yes, my bad. I think keeping the `-u` but setting `${HTTP_PROXY:-}` would be the best way forward. Then we signal that we accept `HTTP_PROXY` to be empty, but if there are any forgotten variables we will be made aware of it and nothing will go under the radar. I will update.